_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,444 @@
+
+The benefits of using open standards and OSS (the code or specification is public, free to reuse and collaboratively adapt and improve; it can be inspected for security issues, is independent from vendors, and gravitates towards interoperable systems) clearly underpin these objectives.
+
+The Policy on Service and Digital makes it clear that working in the open and collaborative working methods will be the norm within the GC's IT community to foster the sharing of code, data and solutions.
+The principal working methods encouraged by this open first strategy are open, inclusive and co-creative.
+
+To drive efficiencies across the GC, the Policy on Service and Digital hinges on the existence of the reusable solutions platform:
+
+ORE ?
+
+> The platform will be built around a catalogue containing reusable building blocks and services, open-source solutions, a corporate architecture, standards, best practices, support and consultancyservices.
+
+We have here a direct link between the Policy on Service and Digital and the open first strategy: the reusable solutions platform proves the benefits of a working culture based on the principles of open standards and OSS.
+
+Another key instrument in the Policy on Service and Digital is the Connecting Europe Facility (CEF);
+here too, the open first strategy provides an actionable link, able to take on board the lessons learned from CEF building blocks to build and foster communities around open source solutions.
+
+The Policy on Service and Digital refers to the vulnerability scanning of digital solutions and services, securing building blocks and patching procedures.
+OSS code can be inspected in detail and is easily included in systematic vulnerability scanning.
+This is another practical link between the Policy on Service and Digital and the open first strategy.
+
+Notes:
+
+Canada has established clear vision and designed it policies to nudge departments towards it.
+By adopting this strategy, ESDC is taking another step in realizing the vision outlined by the governement.
+
+### Sharing and reuse benefits all
+
+Open standards and OSS make it easy to share and reuse solutions, as well as data, information and knowledge.
+This is also in line with the Policy on Service and Digital and the Mandatory Procedures for Enterprise Architecture (EA) Assessment:
+
+- Data should be shared openly by default as per the Directive on Open Government
+- All source code must be released under an appropriate open source software license
+- Leverage and reuse existing solutions, components, and processes
+
+Note:
+
+Working in the open, including software development, helps ensure that the resources used in the development of solutions for Canadians are
+
+This approach also enables further collaboration across the multiple jurisdictions within the country and with private sector as well.
+
+### Contribute to the knowledge society
```

Comment:
This is more of a duty as a government to share with society the work that has been done in the course of its mandate. Transparency, trust, open collaboration.